### PR TITLE
Add example about node label management

### PR DESCRIPTION
**Pull request by flavio**
Add example covering node label management as requested [here](https://github.com/kubernetes-incubator/client-python/issues/240#issuecomment-310497983).

### CODE REVIEW

File: examples/add-node-label.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from pprint import pprint
```
Comment (mbohlool) [inline examples/add-node-label.py]: Add copyright header please.

File: examples/add-node-label.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+from kubernetes import client, config
+
+config.load_kube_config()
+client.configuration.debug = True
```
Comment (mbohlool) [inline examples/add-node-label.py]: We don't need this in the example.

File: examples/remove-node-label.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from pprint import pprint
```
Comment (mbohlool) [inline examples/remove-node-label.py]: We can add this to the previous example. right? in between we can also read the node and check to see if the label is there.

**Comment by flavio** [comment]
I applied all the chances your proposed.

**Comment by mbohlool** [comment]
I think it is good to merge. Please fix pep8 check, more info here: https://travis-ci.org/kubernetes-incubator/client-python/jobs/249944871

Thanks.

**Pull request merged**